README.md

```diff
@@ -11,8 +11,8 @@ Click here to watch a screencast of the KSQL demo on YouTube.
 # Quick Start
 If you are ready to see the power of KSQL, try out these:
 
-- [KSQL Quick Start](/docs/quickstart#quick-start): Demonstrates a simple workflow using KSQL to write streaming queries against data in Kafka.
-- [Clickstream Analysis Demo](/ksql-clickstream-demo#clickstream-analysis): Shows how to build an application that performs real-time user analytics.
+- [KSQL Quick Start][Quick Start]: Demonstrates a simple workflow using KSQL to write streaming queries against data in Kafka.
```

Comment:
Was it an intentional change to move the actual hyperlink to the bottom of the file? (same for two other changed links in here)

Reply:
Yeah - I wanted all the links that would need to be updated on release to be grouped together so its less likely one gets missed.

Reply:
@rodesai i would vote to keep this as-is. especially since the rest of the docs do not follow this convention.

Reply:
Linking to the release-tagged docs from the master README.md, or moving the links down to the bottom of the markdown file?

Reply:
@rodesai i meant, not moving the links to the bottom. :)